src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -61,4 +61,28 @@ public WebDriverWait getWait() {
     public void setWait(final WebDriverWait wait) {
         this.wait = wait;
     }
+
+    /**
+     * Check whether the file is downloaded.
+     * @param fileName expected File name
+     */
+    public void downloadFile(final String fileName) {
+        String downloadFileName = null;
+        boolean fileFound = false;
+        final long sleepTime = 5 * 1000;
```

Comment:
Checkstyle should have complained about magic numbers here

Reply:
I'm reusing the existing code logic as such editor does not give any Checkstyle error